flake8_builtins.py

```diff
@@ -48,7 +48,7 @@ def run(self):
         tree = self.tree
 
         if self.filename == 'stdin':
-            lines = stdin_utils.stdin_get_value().splitlines(True)
+            lines = stdin_utils.stdin_get_value()
             tree = ast.parse(lines)
 
         for statement in ast.walk(self.tree):
```

Comment:
While reporting the bug that this PR fixes (whoops, you beat me to it) I noticed that `tree` is actually unused here. https://github.com/gforcada/flake8-builtins/issues/6#issuecomment-302906438

I think this line needs to be changed to `ast.walk(tree)`.